homeassistant/components/workday/config_flow.py

```diff
@@ -284,7 +299,7 @@ async def async_step_init(
         new_schema = self.add_suggested_values_to_schema(
             schema, user_input or self.options
         )
-
+        LOGGER.debug("Has errors occurred %s", errors)
```

Comment:
Probably should use a different phrase to better distinguish from previous debug message about errors